content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -189,6 +190,17 @@ or `kubeadm upgrade apply`), kubeadm respects the value of `UnversionedKubeletCo
 (during `kubeadm join`, `kubeadm reset`, `kubeadm upgrade ...`), kubeadm attempts to use unversioned ConfigMap name first;
 if that does not succeed, kubeadm falls back to using the legacy (versioned) name for that ConfigMap.
 
+`UpgradeAddonsBeforeControlPlane`
+: This is introduced as a **deprecated** feature gate in v1.28. Before v1.27 (including v1.27), kubeadm will upgrade the 
+addons (including CoreDNS and kube-proxy) immediately during `kubeadm upgrade apply`, regardless of whether there are other 
+control plane instances that have not been upgraded, which may cause compatibility problems. Since v1.28, kubeadm will 
```

Comment:
```suggestion
control plane instances that have not been upgraded, which may cause compatibility problems. Since v1.28, kubeadm defaults to a mode that
```